### PR TITLE
refactor(console): improve ux

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

- reorg code in `EndpointsAndCredentials`
- resolve LOG-9701 new design for expired secrets
- resolve LOG-9702 handle secret name text overflow
- resolve LOG-9704 improve secrets format
- resolve LOG-9705 update console guide to use the new secrets
- resolve PRD-757 use `Never` as the default secret expiration option

## Testing

all issues are locally previewed

## Checklist

- [ ] `.changeset`
- [ ] unit tests
- [ ] integration tests
- [ ] necessary TSDoc comments

### CODE REVIEW

File: packages/console/src/pages/ApplicationDetails/index.tsx [hunk 1]
```diff
@@ -19,21 +21,25 @@ function ApplicationDetails() {
...
+  const isLoading =
+    (!data && !error) ||
+    (!oidcConfig.data && !oidcConfig.error) ||
+    (!secrets.data && !secrets.error);
+  const requestError = error ?? oidcConfig.error ?? secrets.error;
 
   if (isGuideView) {
+    if (!data || !secrets.data) {
+      return <Daisy />;
+    }
```
Comment (gao-sun) [inline packages/console/src/pages/ApplicationDetails/index.tsx:36]: @charIeszhao can you double check if it's good

File: packages/console/src/pages/ApplicationDetails/ApplicationDetailsContent/EndpointsAndCredentials/use-secret-table-columns.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        title: t('application_details.secrets.expires_at'),
+        dataIndex: 'expiresAt',
+        colSpan: 3,
+        render: ({ expiresAt }) => (
+          <span>
+            {expiresAt ? (
+              compareDesc(expiresAt, new Date()) === 1 ? (
+                <Expired expiresAt={new Date(expiresAt)} />
+              ) : (
+                new Date(expiresAt).toLocaleString()
```
Comment (xiaoyijun) [inline packages/console/src/pages/ApplicationDetails/ApplicationDetailsContent/EndpointsAndCredentials/use-secret-table-columns.tsx:71]: We have a component `DateTime` to render the time.

**Review by xiaoyijun — COMMENTED** [review]
Please double check if the modified `CopyToClipBoard` component works fine on other pages.

**Review by charIeszhao — APPROVED** [review]

File: packages/console/src/pages/ApplicationDetails/ApplicationDetailsContent/EndpointsAndCredentials/use-secret-table-columns.tsx (hunk 2 above)
Comment (gao-sun) [reply packages/console/src/pages/ApplicationDetails/ApplicationDetailsContent/EndpointsAndCredentials/use-secret-table-columns.tsx:71]: discussed offline, refactor that component

**Pull request merged**